### ISSUE TITLE
can't change profile picture

### ISSUE THREAD

**Issue by rndt3k** [mgp25/Instagram-API#3043]
# Bug Report

There is an issue with uploading a profile picture. I have tried different pictures/accounts but no luck.  Does anyone know what the problem could be?

### Code
Please post your code relevant to the bug in the section below:
```php
...
$ig = new \InstagramAPI\Instagram($debug, $truncatedDebug);
try {
    $ig->login($username, $password);
} catch (\Exception $e) {
    echo 'Something went wrong: '.$e->getMessage()."\n";
    exit(0);
}
try {
   //$ig->account->changeProfilePicture($photoFilename);

   $photo = new \InstagramAPI\Media\Photo\InstagramPhoto($photoFilename);
   $ig->account->changeProfilePicture($photo->getFile());
   
} catch (\Exception $e) {
    echo 'Something went wrong: '.$e->getMessage()."\n";
}
``` 

### Debug Log
Please post your debug log in the section below; You can enable the debug log by using `new Instagram(true)` instead of `new Instagram()`. Additionally, you can do `\InstagramAPI\Debug::$debugLog = true;` for the debug log to be saved to a file:
```php
POST:  https://i.instagram.com/api/v1/accounts/change_profile_picture/
→ 46.41kB
← 400    58B
RESPONSE: Something is wrong. Please check your input and try again.

Something went wrong: Invalid request options.
```

**Comment by JRoy** [comment]
This was fixed in the private API, please see #2655 for more info

**Comment by moazam1** [comment]
Should we consider this plugin will never update for public? Uploading a photo on instagram is a basic thing and if someone has to pay for the private API to get the updates then it defeats the purpose of open source. Are you going to close this repo in favor of private api?
